### DISCUSSION TITLE
I forgot my ssh password, how do I reset it

### DISCUSSION THREAD

**Discussion by ghost** [community/community#125580] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I forgot my ssh password, how do I reset it

**Comment by davevad93** [comment]
Hi @IsmaelDSC23 ,

In `Windows` and `Linux` OS it's not possible to recover a `SSH Key` password, the only solution would be to create a new one, for more information see [here](https://docs.github.com/en/authentication/troubleshooting-ssh/recovering-your-ssh-key-passphrase?platform=windows).
